e2e/portal/tests/ProjectSettings/FSP configuration/ReconfigureFspConfiguration.spec.ts

```diff
@@ -0,0 +1,79 @@
+import test from '@playwright/test';
+
+import { Fsps } from '@121-service/src/fsps/enums/fsp-name.enum';
+import { FSP_SETTINGS } from '@121-service/src/fsps/fsp-settings.const';
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+
+import FspSettingsPage from '@121-e2e/portal/pages/FspSettingsPage';
+import HomePage from '@121-e2e/portal/pages/HomePage';
+import LoginPage from '@121-e2e/portal/pages/LoginPage';
+import RegistrationsPage from '@121-e2e/portal/pages/RegistrationsPage';
+
+const configuredFsps = [
+  FSP_SETTINGS[Fsps.intersolveVisa].defaultLabel.en,
+  FSP_SETTINGS[Fsps.intersolveVoucherWhatsapp].defaultLabel.en,
+].filter((label): label is string => label !== undefined);
+
+const visaConfiguration = [
+  'Visa debit card',
```

Comment:
and then this one should also come from FSP SETTINGS (plus more occurrences in this and other files)